randovania/games/dread/json_data/Artaria.txt

```diff
@@ -676,7 +676,11 @@ Extra - asset_id: collision_camera_006
           # Spider ceiling wall here. Probably don't need an event for it.
           Spider Magnet or Simple IBJ or Use Spin Boost
   > Door to EMMI Zone Ballspark Hallway
-      Simple IBJ or Use Spin Boost
+      Any of the following:
+          Flash Shift or Speed Booster or Simple IBJ or Use Spin Boost
+          All of the following:
+              # Enemy that you freeze only spawns after White EMMI is defeated.
+              Ice Missile and After Artaria - EMMI First Chase and Stand on Frozen Enemy (Beginner)
```

Comment:
I suggest using the template for Shoot Ice Missile. There could be a time where the main launcher can be shuffled and required, and this could be bugged logic expecting you to go here with ice that you cant use